server/server.go

```diff
@@ -112,3 +114,69 @@ func (s *Server) SumCalories(id string) (int, error) {
 	}
 	return sum, nil
 }
+
+// CheckDB : return a boolean check on the database structure
+func (s *Server) CheckDB() (bool) {
```

Comment:
I don't think `bool` needs top be bracketed

Reply:
Also, just a thought but this function might be better named `CheckDatabaseIntegrity`